### PR TITLE
Modify sort sequence diagram

### PR DESCRIPTION
**Pull request by chewjh1234**

### CODE REVIEW

File: docs/images/SortSequenceDiagram2.png
Comment (Carlintyj) [inline docs/images/SortSequenceDiagram2.png:1]: Left side of the model can include the Logic API with a single lifeline of the Logic

**Review by Carlintyj — CHANGES_REQUESTED** [review]
LGTM! Just some nits to change

File: docs/images/SortSequenceDiagram2.png
Comment (Carlintyj) [reply docs/images/SortSequenceDiagram2.png:1]: Also it should be called ModelManager

Comment (chewjh1234) [reply docs/images/SortSequenceDiagram2.png:1]: @Carlintyj Thank you for spotting these. I have updated the diagrams :D

**Review by Carlintyj — APPROVED** [review]
LGTM!

**Pull request merged**